test_xdb/models/under_test/clone_schema_test.sql

```diff
@@ -0,0 +1,200 @@
+{{config({
```

Comment:
This model shows us how many deltas in tables and views counts in test schemas we have.